presto-orc/src/main/java/com/facebook/presto/orc/writer/BooleanColumnWriter.java

```diff
@@ -176,9 +176,11 @@ public void close()
     private static List<Integer> createBooleanColumnPositionList(
             boolean compressed,
             BooleanStreamCheckpoint dataCheckpoint,
-            Optional<BooleanStreamCheckpoint> presentCheckpoint)
+            Optional<BooleanStreamCheckpoint> presentCheckpoint,
+            Optional<BooleanStreamCheckpoint> inMapCheckpoint)
     {
         ImmutableList.Builder<Integer> positionList = ImmutableList.builder();
+        inMapCheckpoint.ifPresent(booleanStreamCheckpoint -> positionList.addAll(booleanStreamCheckpoint.toPositionList(compressed)));
```

Comment:
The current naming convention for present stream checkpoint is also present checkpoint.
This will make it more verbose. Similarly in the above function as well